modules/function-sourced-from-repository/README.md

```diff
@@ -0,0 +1,73 @@
+# Function Sourced from Repository
```

Comment:
Naming - this name is a bit on the verbose side, would `repository-function` or something similar work, or would shortening the name hide information?

Reply:
+1, I think `repository-function` is better as well.